Flutter/xlsio/working-with-excel-worksheet.md

```diff
@@ -69,6 +69,28 @@ File('Output.xlsx').writeAsBytes(bytes);
 
 {% endhighlight %}
 
+## Highlight Worksheet Tabs
+
+A particular worksheet tab can be highlighted to denote its importance. Tab color can be set through the **tabColor** property, as shown below.
```

Comment:
@KonduruKeerthi  - Please do the below changes.

Title : Worksheet Tab Color

A worksheet can be highlighted with a tab color. Tab color can be set through the **tabColor** property, as shown below.

Reply:
@Mohan2401  - Made the suggested changes